services/grpc/internal/transformations/graph_test.go

```diff
@@ -0,0 +1,32 @@
+package transformations
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// go test graph_objects_test.go graph_objects.go transformations.go methods.go
+// TODO solidify the interface for Call()
+func TestGraphObjects(t *testing.T) {
```

Comment:
TODO: Refactor this to (1) run tests at different integration mocking (i.e. use interfaces so that testing mocks can be easier written) and (2) each test mock provides a kind of up-to-date tutorial.